data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>Maybe a brief explanation of why it would be helpful?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Tbh, I'm feeling cynical rn and don't know if it actually would be that helpful, but I don't want to remove it if it's resolved in later models. Wdyt?</p>

Reply:
<p>Mathematically speaking I don't believe it improves the models since it's a linearly-dependendant feature. We can probably take it out and I can check the models and make the appropriate fixes.</p>

Reply:
<p>I think I removed it and references to it! </p>